pkg/cmd/agent-task/view/view.go

```diff
@@ -98,17 +120,37 @@ func viewRun(opts *ViewOptions) error {
 	var session *capi.Session
 
 	if opts.SessionID != "" {
-		if sess, err := capiClient.GetSession(ctx, opts.SessionID); err != nil {
+		sess, err := capiClient.GetSession(ctx, opts.SessionID)
+		if err != nil {
 			if errors.Is(err, capi.ErrSessionNotFound) {
 				fmt.Fprintln(opts.IO.ErrOut, "session not found")
 				return cmdutil.SilentError
 			}
 			return err
-		} else {
-			session = sess
 		}
+
+		opts.IO.StopProgressIndicator()
+
+		if opts.Web {
+			var webURL string
+			if sess.PullRequest != nil {
+				webURL = fmt.Sprintf("%s/agent-sessions/%s", sess.PullRequest.URL, url.PathEscape(sess.ID))
+			} else {
+				// Currently the web Copilot Agents home GUI does not support focusing
+				// on a given session, so we should just navigate to the home page.
+				webURL = capi.AgentsHomeURL
```

Comment:
💭 I think something like this is what you meant to describe here, since there is indeed a web page for a specific session: 

```suggestion
				// Currently, there is no Agents web UI that supports focusing
				// on a repository. Since we do not have the PR URL required navigate
				// to a specific session, we must navigate to the Agent's home page instead.
				webURL = capi.AgentsHomeURL
```

Reply:
Hmmm nevermind actually, I misread this comment.